ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientImpl.java

```diff
@@ -144,7 +144,7 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
             }
             return ingestionResult;
         } catch (Exception ex) {
-            log.error(String.format("ingestFromStream: Error while ingesting from stream. Error: %s", ex.getMessage()), ex);
+            log.error("ingestFromStream: Error while ingesting from stream. Error: {}", ex.getMessage(), ex);
```

Comment:
Do we need both getMessage and ex?

Reply:
No, we talked about this - SLF4J prints out the message along with the stack

Reply:
sure, it was a mistake from the last merge!